src/client/rsg-components/ComponentsList/ComponentsListRenderer.tsx

```diff
@@ -39,50 +38,55 @@ const styles = ({ color, fontFamily, fontSize, space, mq }: Rsg.Theme) => ({
 });
 
 interface ComponentsListRendererProps extends JssInjectedProps {
-	items: Rsg.Component[];
+	items: Rsg.TOCItem[];
 }
 
 export const ComponentsListRenderer: React.FunctionComponent<ComponentsListRendererProps> = ({
 	classes,
 	items,
+}) => {
+	return (
+		<ul className={classes.list}>
+			{items.map(item => (
+				<ComponentsListSectionRenderer key={item.slug} classes={classes} {...item} />
+			))}
+		</ul>
+	);
+};
+
+const ComponentsListSectionRenderer: React.FunctionComponent<Rsg.TOCItem & JssInjectedProps> = ({
+	classes,
+	heading,
+	visibleName,
+	href,
+	content,
+	shouldOpenInNewTab,
+	selected,
+	open: _open,
```

Comment:
```suggestion
	open: initialOpen,
```

Let's use meaningful names.

Reply:
indeed !!